### PR TITLE
Change benchmark from `tilt_sector` and `input_tilt_sector` to `tilt_subsector` and `input_tilt_subsector` respectively

### LINKED ISSUE #797: Change benchmark from `tilt_sector` to `tilt_subsector`
Hi @kalashsinghal , 

please change the threshold in emission_profile() tilt_sector to tilt_subsector. Let me know if you need further information. And once you get started, could you please also give an indication of how much effort it is? 

Thank you!

cc' @AnneSchoenauer

### PR DESCRIPTION
**Pull request by kalashsinghal**
* Closes #797 

Dear @maurolepore,

As per the new requirement, we have to change the benchmark from `tilt_sector` and `input_tilt_sector` to `tilt_subsector` and `input_tilt_subsector` respectively. Please review this change asap. Thanks!

cc @Tilmon @AnneSchoenauer 

----

TODO

- [x] [Link related issue/PR]([url](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). 
- [x] Describe the goal of the PR. Avoid details that are clear in the diff.
- [x] Mark the PR as draft.
- [x] [Include a unit test](https://code-review.tidyverse.org/reviewer/aspects.html#sec-tests).
- [x] Review your own PR in "Files changed".
- [x] Ensure the PR branch is updated.
- [x] Ensure the checks pass.
- [x] Change the status from draft to ready.
- [x] Polish the PR description to reflect what it ended up doing.
- [x] Polish the PR title as you'd like others to read it from the git log.
- [x] Assign a reviewer.
- [ ] Document user-facing changes in the changelog.

### CODE REVIEW

File: tests/testthat/test-emissions_profile_upstream.R [hunk 1]
```diff
@@ -181,7 +181,7 @@ test_that("at product level, `NA` in a benchmark yields `NA`s only in the corres
   expect_false(is.na(filter(out, clustered == "a")$risk_category))
   expect_true(is.na(filter(out, clustered == "b")$risk_category))
 
-  benchmark <- "input_tilt_sector"
+  benchmark <- "input_tilt_subsector"
   co2 <- example_inputs(
```
Comment (kalashsinghal) [inline tests/testthat/test-emissions_profile_upstream.R:185]: @Tilmon @AnneSchoenauer I have also changed the benchmark from `input_tilt_sector` to `input_tilt_subsector`. This change will affect the `emissions_profile_upstream` indicator. I have done this change to remain aligned with the methodology we are using for `emission_profile` indicator. Should I do this change or not?

Comment (AnneSchoenauer) [reply tests/testthat/test-emissions_profile_upstream.R:185]: Yes, it should also be for the emission_profile_upstream. However, as we don't estimate the upstream yet for the Bundesbank and publishing the data on our webtool it is fine if you just write a reminder ticket and do this at a later stage! Best Anne

Comment (AnneSchoenauer) [reply tests/testthat/test-emissions_profile_upstream.R:185]: Hi Kalash - okay! Let's do it like this :) Please also clarify a bit more what you mean by "estimation" :)

Comment (kalashsinghal) [reply tests/testthat/test-emissions_profile_upstream.R:185]: Hi @AnneSchoenauer, by estimation I mean any kind of analytical assessment that you and Tilman might have to do for `emission_profile_upstream` if "input_tilt_sector" is changed to "input_tilt_subsector". I am not sure if its required. If its not required then I will not create any ticket as a reminder. Please conform if you guyz need a ticket!

Comment (AnneSchoenauer) [reply tests/testthat/test-emissions_profile_upstream.R:185]: I think its fine without a ticket :) Thanks!

File: R/example_dictionary.R [hunk 2]
```diff
@@ -96,7 +96,7 @@ example_id <- function() {
            18L,        4L,        2L,
            13L,        1L,        2L,
             1L,        1L,        3L,
-           16L,        1L,        3L,
+           17L,        1L,        3L,
```
Comment (maurolepore) [inline R/example_dictionary.R:99]: Thanks. This seemed to belong to a separate issue/PR. Now `example_products()` has `tilt_subsector` (#803).

**Review by maurolepore — COMMENTED** [review]
Thanks @kalashsinghal 

* ml01. I need  a little time to see if we can instead remap "tsector" to "tilt_subsector". In principle it should be possible and simpler.

* ml02. In the original issue by @Tilmon I see he refers only to `emissions_profile()` but this PR seems to also affect `emissions_profile_upstream()`.

* ml03. See #803

**Comment by Tilmon** [comment]
@maurolepore 

> ml02. In the original issue by @Tilmon I see he refers only to emissions_profile() but this PR seems to also affect emissions_profile_upstream().

Yes, it does also affect `emission_profile_upstream()`, but we don't need this change now. We won't use upstream results right now. I.e., if implementing it for the `emission_profile_upstream()` costs extra time, we should leave it out for now and change it at a later stage. For now, we only need results for the indicators that are not upstream.

**Comment by maurolepore** [comment]
Thanks @Tilmon 

It's actually less work to do it for both than for one indicator alone.

**Review by maurolepore — APPROVED** [review]
LGTM, 

Thanks @kalashsinghal. 

I pushed a commit to revert the change to R/example_dictionary.R since that change is already in `main` (9dc8e3a17babad55ba8c3ffdbcb0d39130ebfb79).

Note that @Tilmon confirmed that we need the same on the *upsteam indicator. Not sure if you need to follow up.

I experimiented a bit with the re-mapping of the alias but I think this PR is good to go.

Please remember to update NEWS or let me know when you merge and I'm happy to do it.

**Pull request merged**